examples/simple_xo/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "simple_xo"
+version = "0.1.0"
+authors = ["Swetha Repakula <srepaku@us.ibm.com>"]
```

Comment:
Make this match the copyright holder(s). This is an active topic of discussion, as to what we should put here, but that's the approach used in the other Cargo.toml file currently.